src/main/java/org/lightcouch/CouchDbClient.java

```diff
@@ -205,6 +206,8 @@ private PoolingHttpClientConnectionManager createConnectionManager(
 
 		if("https".equals(props.getProtocol())) {
 			if (props.isSSLAuthenticationDisabled()) {
+				// No SSL authentication.
+				// No need for a custom SSLSocketFactory in this case.
```

Comment:
If it doesn't make sense to both disable SSL authentication and set a custom SSL factory this needs at least documenting in the public documentation, if not including checks for in the setters in `ConnectOptions`.